Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.service.opg.digital/opguk/php-fpm
+FROM registry.service.opg.digital/opguk/php-fpm:0.1.207
```

Comment:
Why are we locking to a version here? I was under the impression we removed this so that when built latest stable would always be pulled in to make sure security updates etc where applied